### PR TITLE
task/224 give more complete information with de Info buttons.

### LINKED ISSUE #224: Rewrite Info text for DisplayImageSet and MetadataEditing fragments

### PR DESCRIPTION
**Pull request by NicolasVial**
This PR : 

- Now have more complete info texts about general infos + possible actions.
- i also fixed a bug in this PR because I did in 1 line of code (it's a dirty hack but I didn't find a way to do smth better). The bug was that the app crash if we long click on a picture in ImageSetFragment and then go back with the back arrow.
- un-ignore a test that is now fixed :)

### CODE REVIEW

**Review by wattsecs — CHANGES_REQUESTED** [review]
~~I think this pr should also remove the long-press info because it requires too high of an Android version and is now obsolete~~
Please ignore this comment, I got confused... Sorry

**Review by wattsecs — APPROVED** [review]

File: app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/view/dataset_editing/MetadataEditingFragmentTest.kt [hunk 1]
```diff
@@ -177,7 +175,7 @@ class MetadataEditingFragmentTest {
             navigateToCreateDatasetFragment()
             onView(withId(R.id.categories_editing_menu_info)).perform(click())
             waitFor(1) // increase if needed
```
Comment (martin-lenweiter) [inline app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/view/dataset_editing/MetadataEditingFragmentTest.kt:177]: Would be nice to also test that the alert dialog is actually displayed. (I think I did test this in the learningSettingsFragment, so have a look there #235)

**Review by martin-lenweiter — CHANGES_REQUESTED** [review]
Small improvement needed in one test, other than that all good

File: app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/view/dataset_editing/MetadataEditingFragmentTest.kt (hunk 1 above)
Comment (NicolasVial) [reply app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/view/dataset_editing/MetadataEditingFragmentTest.kt:177]: Oh yes ty you're right. i'll add it here and in ImageSetTest too :=)

**Review by martin-lenweiter — APPROVED** [review]

**Pull request merged**